### ISSUE TITLE
Account Management View

### ISSUE THREAD

**Issue by kernelkind** [damus-io/android#114]
A view should exist which displays the user's accounts. This is where the user can see, add, and remove accounts from the app

**Comment by kernelkind** [comment]
related to #111

**Comment by kernelkind** [comment]
> This is great, and will be a killer notedeck feature.
> 
> Let me push back and ask: should multiple accounts be a part of the first [notedeck/android milestone 0.1](https://github.com/damus-io/android/milestone/1)?

I don't know if it should be part of 0.1, but I believe we should at least be thinking of features with multiple accounts in mind so we don't need big re-writes in the future

**Comment by alltheseas** [comment]
Thanks for confirming, and thanks for looking ahead. I appreciate this and am onboard with this approach 🙏

**Comment by jb55** [comment]
yes ideally anything we build out should not be a single keyset and should plan for multiple keys.
